flathub.json

```diff
@@ -0,0 +1,4 @@
+{
+	"only-arches": ["x86_64"],
+	"automerge-flathubbot-prs": true
```

Comment:
```suggestion
	"only-arches": ["x86_64"]
```

automerge is not allowed.

Reply:
Should https://docs.flathub.org/docs/for-app-authors/maintenance/#automatically-merging-prs be updated with a disclaimer then?